prowler/providers/azure/services/app/app_function_ftps_deployment_disabled/app_function_ftps_deployment_disabled.metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+  "Provider": "azure",
+  "CheckID": "app_function_ftps_deployment_disabled",
+  "CheckTitle": "Ensure that FTP and FTPS deployments are disabled for Azure Functions to prevent unauthorized access and data breaches.",
```

Comment:
Why FTPS?

Reply:
It is not a good practice use the FTP protocol in general to deploy azure functions, it is explained in metadata. It is not recommended by Microsoft any deploy that needs manually sync trigger [ref](https://learn.microsoft.com/en-us/azure/azure-functions/functions-deployment-technologies?tabs=windows#trigger-syncing). In regular web apps we have only FTP because CIS required in this way but we could apply the same logic and does not recommend FTPS too.